src/role_utils.rs

```diff
@@ -172,6 +173,8 @@ fn config_schema_default() -> serde_json::Value {
 pub struct Role<T> {
     #[serde(flatten)]
     pub config: CommonConfiguration<T>,
+    #[serde(default)]
+    pub role_config: RoleConfig,
     pub role_groups: HashMap<String, RoleGroup<T>>,
```

Comment:
Let's space this out.

```suggestion
    #[serde(flatten)]
    pub config: CommonConfiguration<T>,
    
    #[serde(default)]
    pub role_config: RoleConfig,
    
    pub role_groups: HashMap<String, RoleGroup<T>>,
```